inc/flatbuffers/CalcHistory_generated.h

```diff
@@ -0,0 +1,169 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
Don't include auto-generated files in the repo. The generation should be part of the cmake or compile process. When I did this, I think I made a custom target for the flatbuffer which will perform the generation and added that target as a dependency of the main calculator executable.